notebooks/growth_model_classification.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>maybe we should reword it to: <code>... differentiate customers based on their growth potentials and identify the top most important customers that the business should be aware of.</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>